src/css/global.css

```diff
@@ -96,3 +96,14 @@ a:active {
 		text-decoration: underline;
 	}
 }
+
+.u-text-white a:link,
+.u-text-white a:visited,
+.u-text-whit a:hover,
+.u-text-white a:active {
+	color: var(--color-purple-200);
+
+	&:hover {
+		filter: brightness(0.75);
+	}
+}
```

Comment:
It looks like there's a _tiny_ typo here, `.u-text-whit a:hover`

I think I might also rewrite this slightly, for readability simply because :hover isn't a pseudoclass that exists on a:visited: 

```
.u-text-white a:link,
.u-text-white a:visited,
.u-text-white a:hover,
.u-text-white a:active {
	color: var(--color-purple-200);
}

.u-text-hover {
  filter: brightness(0.75);
}

```

Reply:
One alternative approach here is to scope the link color to a component itself, rather than a utility class. For example, these examples are from the footer (which uses scoped css.)

```
	.footer__textLink {
				color: inherit;
				text-decoration: none;

				&:hover {
					color: var(--color-yellow);
				}
			}

			.footer__textLink--emphasis {
				text-decoration: underline;
			}
```

Reply:
> Utility classes like .u-text-white are also intended to do one thing only

Cool, I will add a separate utility class.

> I think I might also rewrite this slightly, for readability simply because :hover isn't a pseudoclass that exists on a:visited

Cool, I just copied and pasted the [`a` tag css from global.css](https://github.com/harvard-lil/capstone-static/blob/main/src/css/global.css#L87-L98). No opinions here!

Reply:
Thank you for calling that out! I can create a lil Linear issue to clean that up, that's an obvious brain fart on my part